EnglishFilter/sections/css_extended.txt

```diff
@@ -1,6 +1,8 @@
 !
 ! Extended CSS rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/5083
+uvnc.com#?#.sp-module:contains(adsbygoogle)
```

Comment:
why not `:has()`? `:contains()` is slower.

Reply:
Will change to `uvnc.com#?##sp-left > div.sp-column > div.sp-module:has(.adsbygoogle)`